frontend/src/pages/ApprovedActivityReport/index.scss

```diff
@@ -20,7 +20,8 @@
 
 @media print {
   .smart-hub-offset-nav {
-    max-width: none;
+    margin-top: 0 !important;
```

Comment:
It is my understanding that we should avoid !important at all costs. Is there any way we can get around using it?

Reply:
unfortunately, no - the margin-top is _inlined_ dynamically based on the height of the header + alerts, and the only way to override inline styles is with important. they are both not recommended but I think they are the best thing to do in this case. luckily, using important in a print style does not usually lead the the same problems as using it for regular application stuff